lib/wings/active_fedora_converter.rb

```diff
@@ -62,26 +64,22 @@ def attributes
     ##
     # @return [ActiveFedora::Base]
     def convert
-      active_fedora_class.new(normal_attributes).tap do |af_object|
-        af_object.id = id unless id.empty?
-        normal_attributes.each_key { |key| af_object.send(:attribute_will_change!, key) }
-        add_access_control_attributes(af_object)
-        convert_members(af_object)
-        convert_member_of_collections(af_object)
-        convert_files(af_object)
+      instance.tap do |af_object|
+        af_object.id ||= id unless id.empty?
+        apply_attributes_to_model(af_object)
       end
     end
 
     def active_fedora_class
-      @active_fedora_class ||=
+      @active_fedora_class ||= # cache the class at the instance level
```

Comment:
this cache was already introduced, but this bit of documentation got left out. i think it's nice to call it out as wanted, since there is also a class level cache.